README.rst

```diff
@@ -1832,9 +1832,22 @@ Choose from the list of available rules:
 
   Use ``null`` coalescing operator ``??`` where possible. Requires PHP >= 7.0.
 
-* **trailing_comma_in_multiline_array** [@Symfony, @PhpCsFixer]
+* **trailing_comma_in_multiline** [@Symfony, @PhpCsFixer]
```

Comment:
maybe separate rules would be more readable in this case?
```
'trailing_comma_in_multiline_array' => true,
'trailing_comma_in_multiline_function_arguments' => true,
```
versus
```
'trailing_comma_in_multiline' => [
    'where' => ['arrays', 'functions'],
]
```
I'd never have guessed the "where" option name without looking into docs (if that is any argument).
Great work implementing it, though.

Reply:
I agree with this, just because I see a lot of ifs everywhere, maybe better to separate it into 2 fixer classes

Reply:
Nah, let's not have 2 fixers (and in PHP 8 we would have one more) doing almost the same.

We don't have `visibility_required_property`, `visibility_required_method` and `visibility_required_const`, but single fixer.

> I'd never have guessed the "where" option name without looking into docs

@Wojciechem you'd never have guessed any fixer name either without looking into docs, would you? Docs are to look into them ;)

> because I see a lot of ifs everywhere

@kunicmarko20 there is only 1 if for each option, other ifs are to have the fixer working